### PR TITLE
Add discourse link

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: docs-src/contact.md [hunk 1]
```diff
@@ -2,9 +2,7 @@
 layout: default
 ---
 
-# Contact us
-
-There are multiple ways to contact directly `Gtk-rs` developers:
+For discussing general GTK questions or how to use the bindings go to [discourse.gnome.org](discourse.gnome.org). However, if you want to contact the `Gtk-rs` developers directly, there are multiple ways:
 
 # IRC
```
Comment (sdroege) [inline docs-src/contact.md:8]: Below for GitHub, also add that creating issues is for bug reports or feature requests and not general support (for that there's discourse, matrix/irc).

File: docs-src/contact.md [hunk 2]
```diff
@@ -18,4 +16,4 @@ You can connect using: **[#rust:gnome.org][]**.
 
 # Github
 
-Of course, you can also just open an issue on our [repositories](https://github.com/gtk-rs) on github.
+Of course, you can also just open an issue on our [repositories](https://github.com/gtk-rs) on github. However, please keep in mind that issues on `Gtk-rs` should be for bug reports or feature requests and not general support.
```
Comment (sdroege) [inline docs-src/contact.md:19]: "For general support use the GNOME Discourse or IRC/Matrix".

File: docs-src/contact.md [hunk 3]
```diff
@@ -2,9 +2,7 @@
 layout: default
 ---
 
-# Contact us
-
-There are multiple ways to contact directly `Gtk-rs` developers:
+For discussing general GTK questions or how to use the bindings go to [discourse.gnome.org](discourse.gnome.org). However, if you want to contact the `Gtk-rs` developers directly, there are multiple ways:
```
Comment (sdroege) [inline docs-src/contact.md:5]: Maybe add a headline: `# GNOME Discourse`?

**Comment by GuillaumeGomez** [comment]
Updated!

**Review by sdroege — APPROVED** [review]

**Pull request merged**